FINGERPRINT_Toolbox_Inventory.adoc

```diff
@@ -34,6 +37,7 @@ a|Commercial fingerprint scanner
 * 1000 dpi minimum image resolution
 * Rolled images preferred
 * Touch/slap images acceptable
+a|
```

Comment:
Let's get examples of easily-accessible 1000 PPI scanners from @gregott in here.

Reply:
I completely agree, but didn't know if we should have them added in this PR or if they could be added later once this is approved (since this is more about the formatting with my two examples added) than it is about the actual set of examples which @gregott probably needs to provide.